README.md

```diff
@@ -75,6 +77,18 @@ generic with minimal need for project-specific behavior.
 [Docker]: https://docs.docker.com/install/overview/
 
 
+###2.1.1 git-secrets
+
+[git-secrets](https://github.com/awslabs/git-secrets) helps to prevent secrets
+(passwords, credentials, etc.) from being submitted into a git repository.
+Follow the [install instructions](https://github.com/awslabs/git-secrets#installing-git-secrets)
+to enable and configure git-secrets in all existing local Azul repositories.
+
+It is also recommended to follow the steps in the [advanced configuration](https://github.com/awslabs/git-secrets#advanced-configuration)
```

Comment:
The source would look cleaner if you converted the inline links to [reference-style](https://www.markdownguide.org/basic-syntax/#reference-style-links). That would also make it easier to inline the section into the bullet.